### PR TITLE
2023 Release Notes

### PR DESCRIPTION
**Pull request by cpaulgilman**
Release notes for Fall 2023 release.

To view as html: https://htmlpreview.github.io/?https://github.com/NREL/SAM/blob/2023-release-notes/doc/releasenotes.html

**Do not merge until SAM and SSC version numbers are updated and installation files are posted on SAM website Downloads page**

This gh-pages branch is a special documentation branch that does not contain any code. Release notes are automatically published at https://nrel.github.io/SAM/doc/releasenotes.html. SAM website redirect points to that URL: https://sam.nrel.gov/releasenotes.

### CODE REVIEW

File: doc/releasenotes.html [hunk 1]
```diff
@@ -59,6 +59,151 @@ <h2>CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical Tr
...
+
+<h2>Hybrid Power Systems</h2>
+
+<p>This major new feature adds configurations for hybrid power systems. (<a href="https://github.com/NREL/SAM/pull/1448">SAM 1448</a> and <a href="https://github.com/NREL/SAM/pulls?q=is%3Apr+milestone%3A%22SAM+Fall+2023+Release%22+label%3A%22dev+bug%22+is%3Aclosed">multiple subsequent pull requests</a>, <a href="https://github.com/NREL/ssc/pull/1051">SSC 1051</a>)</p>
+
+<p>Hybrid power systems combine photovoltaic (PVWatts or Detailed PV), Wind Power, Fuel Cell, and Generic System models for power generation with the Standalone Battery model for electricity storage. Battery dispatch responds to the sum of power from the PV, wind, fuel cell and/or generic systems in each time step. The hybrid system performance models are availailabe with either the Single Owner for front-of-meter applications or the Third Party Host / Developer financial model for behind-the-meter applications. As is the case for all of SAM's models, SAM assumes the hybrid system is connected to the grid. The new hybrid configurations are available for the following combinations of performance models:</p>
```
Comment (mjprilliman) [inline doc/releasenotes.html:86]: typo on 'available' (availailabe)

File: doc/releasenotes.html [hunk 2]
```diff
@@ -59,6 +59,151 @@ <h2>CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical Tr
...
+<h1 id="2023-12-xx">SAM 2023.12.xx, SSC xxx: December xx, 2023</h1>
+
+<p>This version introduces a set of new hybrid power system models and a new molten salt linear Fresnel models for electric power generation and industrial process heat (IPH) applications.</p>
+
+<p>For a complete list of code revisions associated with this version, see the <a href="https://github.com/NREL/SAM/pulls?q=is%3Apr+milestone%3A%22SAM+Fall+2023+Release%22+is%3Aclosed">SAM</a> and <a href="https://github.com/NREL/ssc/pulls?q=is%3Apr+milestone%3A%22SAM+Fall+2023+Release%22+is%3Aclosed">SSC</a> GitHub repositories.  Highlights include:</p>
+
+<ul>
+<li>New SAM logo and icon.</li>
+<li>New hybrid power models with different combinations of the photovoltaic (PVWatts and Detailed PV), Wind Power, Fuel Cell, and Generic System models with battery storage for the Single Owner and Third Party Host / Developer financial models.</li>
+<li>New concentrating solar power (CSP) and industrial process heat (IPH) model for molten salt linear Fresnel systems.</li>
```
Comment (mjprilliman) [inline doc/releasenotes.html:71]: I'd consider adding the self-consumption dispatch option as a key bullet point here.

File: doc/releasenotes.html [hunk 3]
```diff
@@ -59,6 +59,151 @@ <h2>CATEGORY: General, Solar, Detailed Photovoltaic, Wind Power, CSP Physical Tr
...
+<li>Remove "Calculate Load Data" option from Electric Load page for Standalone Battery / Residential configurations because it requires solar resource data that is not available. (<a href="https://github.com/NREL/SAM/pull/1467">SAM 1467</a>)</li>
+<li>Set default values of all availability loss inputs to zero for battery configurations. (<a href="https://github.com/NREL/SAM/pull/1463">SAM 1463</a>)</li>
+<li>Show rear irradiance values in loss diagram for PV-battery configurations. (<a href="https://github.com/NREL/SAM/pull/1422">SAM 1422</a>)</li>
+<li>Fix issue for PV-smoothing dispatch that caused discrepancy between PV-smoothing and battery state of charge values. (<a href="https://github.com/NREL/ssc/pull/1085">SSC 1085</a>)
+<li>Fix battery does not discharge for peak shaving dispatch for small batteries. (<a href="https://github.com/NREL/ssc/pull/1208">SSC 1208</a>)</li>
```
Comment (mjprilliman) [inline doc/releasenotes.html:155]: 'Fix battery not discharging' maybe?

**Comment by janinefreeman** [comment]
My only comment would be to add the self-consumption dispatch to the list of highlights, as I think that will be a popular option. Otherwise looks good here!

**Review by janinefreeman — APPROVED** [review]

**Pull request merged**